src/static/js/ace2_inner.js

```diff
@@ -2472,20 +2472,10 @@ function Ace2Inner(){
       }
     }
 
-    if (selectionAllHasIt)
-    {
-      ifAttribIsFormattingUpdateStyleButtonState(attributeName, false); // italic, bold, ...
-      documentAttributeManager.setAttributesOnRange(rep.selStart, rep.selEnd, [
-        [attributeName, '']
-      ]);
-    }
-    else
-    {
-      ifAttribIsFormattingUpdateStyleButtonState(attributeName, true); // italic, bold, ...
-      documentAttributeManager.setAttributesOnRange(rep.selStart, rep.selEnd, [
-        [attributeName, 'true']
-      ]);
-    }
+
+    var attributeValue = selectionAllHasIt ? '' : 'true';
+    documentAttributeManager.setAttributesOnRange(rep.selStart, rep.selEnd, [[attributeName, attributeValue]]);
+    if (attribIsFormattingStyle(attributeName)) updateStyleButtonStateWhenAttribIsFormatting(attributeName, !selectionAllHasIt); // italic, bold, ...
```

Comment:
styling: can you use curly braces on the `if` as we have on the other `if`s on this module, please?